libqtile/widget/volume.py

```diff
@@ -156,10 +164,7 @@ def _update_drawer(self):
             elif self.volume >= 80:
                 self.text = "\U0001f50a"
         else:
-            if self.volume == -1:
-                self.text = "M"
-            else:
-                self.text = "{}%".format(self.volume)
+            self.text = self.format.format(volume=self.volume, mute=self.mute)
```

Comment:
Oh, hang on... the Pulse volume widget inherits the `Volume` widget so I wonder if this breaks it. Let's see what happens in the tests!